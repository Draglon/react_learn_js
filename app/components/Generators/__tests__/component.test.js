import React from 'react';

import GeneratorsComponent from '../component';

describe('Generators component matches snapshot', () => {
  const component = shallow(<GeneratorsComponent />);

  it('with default props', () => {
    expect(component).toMatchSnapshot();
  });
});
