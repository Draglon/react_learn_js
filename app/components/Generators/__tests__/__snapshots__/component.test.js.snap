// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Generators component matches snapshot with default props 1`] = `
<Layout
  className="layout"
>
  <div
    className="top-margin"
  >
    <Title>
      Генераторы
    </Title>
  </div>
  <div
    className="example"
  >
    <h1
      className="example__title"
    >
      Пример 1:
    </h1>
    <h2
      className="example__subtitle"
    >
      Клендарь
    </h2>
    <p
      className="example__text"
    />
    <div
      className="example__html-code"
    >
      <div
        className="calendar"
      >
        <ul
          className="calendar__list"
        >
          <li
            className="calendar__list"
          >
            <div>
              Number
            </div>
          </li>
        </ul>
      </div>
      <button
        className="example__btn-play"
        type="button"
      >
        Play
      </button>
    </div>
  </div>
</Layout>
`;
